docs/hub/webhooks.md

```diff
@@ -4,13 +4,13 @@
 
 Webhooks are a foundation for MLOps related features. You can use them to auto-convert models, build community bots, or build CI/CD for your models, datasets, and Spaces.
 
-They allow you to listen for new changes on specific repos or repos belonging to particular users/organizations (not just your repos, but any repo!).
+They allow you to listen for new changes on specific repos or to all repos belonging to particular users/organizations (not just your repos, but any repo).
 
-You can create new webhooks and edit exiting ones in your webhooks [settings](https://huggingface.co/settings/webhooks):
+You can create new webhooks and edit existing ones in your webhooks [settings](https://huggingface.co/settings/webhooks):
 
 ![Settings of an individual webhook](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/webhook-settings.png)
 
-Webhooks can watch for repos updates, Pull requests, discussions, and new comments. It's even possible to create a Space to react to your webhooks!
+Webhooks can watch for repos updates, Pull requests, discussions, and new comments. If you need to host your webhook, it's of course possible to use a Space to host it.
```

Comment:
for me, the previous version "It's even possible to create a Space to react to your webhooks!" was clearer

Reply:
yes, feel free to revert + merge then (let's wait for @coyotte508 review maybe)

Reply:
i'm split!

Reply:
ok will revert that sentence.

cc @osanseviero fyi too